docs/components/campaigns.rst

```diff
@@ -855,3 +855,125 @@ The Campaign Engine then dispatches the Decision Event's ``eventName`` where lis
     :param mixed $channelId: ID of the Channel entity.
 
     :returntype: void
+
+
+Exporting a campaign
+********************
+
+Mautic's export functionality provides campaign structures in formats that can be easily imported back into Mautic with the import feature. This helps ensure your marketing assets remain portable and preserved, expecially if you are working across many Mautic instances and domains.
+
+Mautic exports all data related to your campaign to a structured JSON file, complete with any files and assets required. The resulting files are archived to a zip file that is saved to your browser.
+
+The export command can be called from the command line or via the API as well as in the Mautic UI.
+
+
+Exporting via Command line
+==========================
+
+First, copy the campaign zip file into the correct place to be imported into Mautic.
+
+.. code-block:: bash
+
+   docker cp ./campaign_data.zip ddev-mautic-web:/tmp/entity_data.zip
+
+Then, run the import command:
+
+.. code-block:: bash
+
+   bin/console mautic:entity:import --entity=campaign --file=/tmp/entity_data.zip --user=*{<user_id>}*
+
+Options:
+
+=====================  ============================================================
+Option                 Description
+=====================  ============================================================
+``--entity=campaign``  Specifies the entity type being imported (e.g. campaign)
+``--file``             Path to the file you want to import (ZIP)
+``--user=``            User ID to associate with the import process (e.g. 1 (admin))
+=====================  ============================================================
+
+Exporting via API
+=================
+
+Endpoint:
+``GET https://{*your-mautic-domain*}/api/campaigns/export/{*<campaign_id>*}``
+
+The final part of the URL, ``{*<campaign_id>*}``, specifies the ID of the campaign to be exported.
+
+Headers:
+
+=================  ===========================
+Header             Description
+=================  ===========================
+``Authorization``  Bearer *<access_token>*
+=================  ===========================
```

Comment:
There is an API authorization section for this. It doesn't have to be mentioned for every API endpoint doc.